qtconsole/base_frontend_mixin.py

```diff
@@ -144,8 +144,12 @@ def from_here(self, msg):
 
     def include_output(self, msg):
         """Return whether we should include a given output message"""
-        if self._hidden:
-            return False
+        if msg['parent_header']:
+            # If parent message is from hidden execution, do not include
```

Comment:
```suggestion
            # If parent message is from hidden execution, don't include it.
```

Small grammar improvement to this comment.